gateway-server/src/test/java/org/apache/knox/gateway/services/topology/DefaultTopologyServiceTest.java

```diff
@@ -177,6 +181,21 @@ public void testGetTopologies() throws Exception {
     }
   }
 
+  /**
+   * Set the static GatewayServices field to the specified value.
+   *
+   * @param gws A GatewayServices object, or null.
+   */
+  private void setGatewayServices(final GatewayServices gws) {
+    try {
+      Field gwsField = GatewayServer.class.getDeclaredField("services");
+      gwsField.setAccessible(true);
+      gwsField.set(null, gws);
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
Not sure you need the catch here? Since it's a test probably better to throw it?

Reply:
+1